Build/Projects/FrameworkReferences.definition

```diff
@@ -25,6 +29,7 @@
         Name="OpenTK"
         Path="ThirdParty/Dependencies/OpenTK.dll" />
     </Service>
+    <Reference Include="MonoGame.Framework.Core.Include" />
```

Comment:
That will pull in the `MonoGame.Framework.Core.Include` include project because the `Linux` platform isn't using PCLs here.

I'm personally still extremely reserved about making the Linux platform reference PCLs since this is still a huge mess across distros at the moment; the Linux platform is the reason Protobuild warns so strongly against using them, because the issues they cause can manifest themselves on end-user machines or developer machines, not build servers.  Debian for example doesn't even package the PCL reference assemblies (along with an ancient version of Mono presumably), so users need to install these manually if a PCL is being targeted.

But @cra0zy seems to think it's fine so I don't know... but if I do run into issues relating to PCLs on Linux in MonoGame I will file issues here since it's not practical to be asking end-users to manually download ZIPs and install them into /usr/lib/mono or where-ever they're meant to go.